Sources/MapboxMaps/Location/Pucks/Puck2D.swift

```diff
@@ -17,14 +17,19 @@ public struct Puck2DConfiguration: Equatable {
     /// The size of the images, as a scale factor applied to the size of the specified image.
     public var scale: Value<Double>?
 
+    /// Flag determining if the horizontal accuracy ring should be shown arround the Puck. default value is false
+    public var showAccuracyRing: Bool
```

Comment:
I think we'd typically say `shows` instead of `show`. @nishant-karajgikar do you agree?

Reply:
updated, I have seen this trend in our code